src/aggregation/mod.rs

```diff
@@ -518,12 +521,14 @@ mod tests {
                 "field": "score",
                 "interval":  70.0,
                 "offset": 3.0,
+                "keyed": false
             },
             "aggs": {
                 "bucketsL2": {
                     "histogram": {
                         "field": "score",
-                        "interval":  70.0
+                        "interval":  70.0,
+                        "keyed": false
```

Comment:
```suggestion
```

Reply:
Oh sorry missed those, fixed in https://github.com/quickwit-oss/tantivy/pull/1424/commits/9b6b60cc2bd66c825fc8cbcc01851cba01601027